libs/datasets/combined_datasets.py

```diff
@@ -85,6 +86,7 @@
     CommonFields.ICU_TYPICAL_OCCUPANCY_RATE: [],
     CommonFields.POSITIVE_TESTS: [CDSDataset, CovidTrackingDataSource],
     CommonFields.NEGATIVE_TESTS: [CDSDataset, CovidTrackingDataSource],
+    CommonFields.CONTACT_TRACERS_COUNT: [TestAndTraceData],
```

Comment:
Can you add this to the latest values dataset above as well?

Reply:
Done